src/clj_puppetdb/http.clj

```diff
@@ -1,11 +1,12 @@
 (ns clj-puppetdb.http
-  (:require [cemerick.url :refer [url-encode map->query]]
+  (:require [cemerick.url :refer [map->query]]
             [cheshire.core :as json]
             [clj-puppetdb.http-util :as util]
+            [clj-puppetdb.query :as q]
             [clj-puppetdb.schema :refer [Client]]
             [clj-puppetdb.util :refer [file?]]
```

Comment:
raynes/fs already provides this method and we already have a dependency on it. We should delete that util namespace.